hoprd-node-1.yaml

```diff
@@ -22,48 +22,48 @@ spec:
       requests:
         cpu: 750m
         memory: 256Mi
-    startupProbe: |
-      exec:
-        command:
-          - sh
-          - '-c'
-          - >-
-            curl -s http://localhost:3001/api/v2/node/info -H
-            "X-Auth-Token: $HOPRD_API_TOKEN" | jq -r .connectivityStatus |
-            grep -q Green
-      initialDelaySeconds: 5
-      timeoutSeconds: 5
-      periodSeconds: 30
-      successThreshold: 1
-      failureThreshold: 60
-    livenessProbe: |
-      exec:
-        command:
-          - sh
-          - '-c'
-          - >-
-            curl -s http://localhost:3001/api/v2/node/info -H
-            "X-Auth-Token: $HOPRD_API_TOKEN" | jq -r .connectivityStatus |
-            grep -q Green
-      initialDelaySeconds: 5
-      timeoutSeconds: 5
-      periodSeconds: 10
-      successThreshold: 1
-      failureThreshold: 5
-    readinessProbe: |
-      exec:
-        command:
-          - sh
-          - '-c'
-          - >-
-            curl -s http://localhost:3001/api/v2/node/info -H
-            "X-Auth-Token: $HOPRD_API_TOKEN" | jq -r .connectivityStatus |
-            grep -q Green
-      initialDelaySeconds: 5
-      timeoutSeconds: 5
-      periodSeconds: 10
-      successThreshold: 1
-      failureThreshold: 5
+    # startupProbe: |
```

Comment:
Intentional?

Reply:
Yes, this is intentional for testing purposes. Everytime that I need new testing scenarios I modify this file